onboarding.md

```diff
@@ -0,0 +1,33 @@
+# Onboarding
+
+Welcome to Hypha! This page is supposed to help those getting situated in our organization.
+
+## Contacting Hypha 
+
+We also have the following communication and contact methods:
+
+- :telephone_receiver: +1-437-887-6936 (We have specific guides for [Accessing][accessing-vm] and [Managing][managing-vm] Voicemail)
+- :mailbox: #1308-1403 Royal York Road, Etobicoke, Ontario  M9P 0A1
+- :link: [hypha.coop](https://hypha.coop)
+- :e-mail: hello@hypha.coop
+
+## Our Virtual Office
+
+In place of a physical meeting place we have virtual office comprised of the following spaces:
+
+- Matrix Chat
+- Loomio
+- GitHub Organization
```

Comment:
Would GH task tracker be a more opportune link than the GitHub org? Or maybe both?

Reply:
Also, thoughts on linking them?

Reply:
I think org more appropriate here, and maybe to tools page after as Other tools